src/lwc/rd2EditPaymentInformationModal/rd2EditPaymentInformationModal.html

```diff
@@ -0,0 +1,76 @@
+<template>
+    <div>
+        <section 
+            role="dialog"
+            tabindex="-1"
+            aria-labelledby="modal-heading-01"
+            aria-modal="true"
+            aria-describedby="modal-content-id-1"
+            class="slds-modal slds-fade-in-open slds-modal_small"
+            onkeyup={handleKeyUp}>
+            <div class="slds-modal__container">
+                <header class="slds-modal__header slds-clearfix">
+                    <lightning-button-icon
+                        class="slds-modal__close"
+                        title="Close"
+                        icon-name="utility:close"
+                        icon-class="slds-button_icon-inverse"
+                        onclick={handleClose}
+                        onkeydown={handleClosedButtonTrapFocus}
+                        data-id="closeButton">
+                    </lightning-button-icon>
+                    <h2
+                        class="slds-text-heading_medium slds-text-heading--medium slds-hyphenate header-string slds-p-top_large slds-p-horizontal_medium" id="modal-heading-01">
```

Comment:
Not sure I understand why both:   slds-text-heading_medium slds-text-heading--medium